WashU-IT-RIS/coldfront-wustl-fork
# ITDEV-35795 - Fixes to Bug Stripping Parent ACL Entries and Change Owner permissions

Previously, running Reset ACLs on the child sub-allocation would remove the `storage-<parent alloc-rw/ro` group entries from files in the child sub-allocation.  In addition, running Reset ACLs resulted in the OWNER entries losing their CHANGE_OWNER (lowercase o) permissions.  This PR fixes both those issues.

Test Instructions:

1. Set up QA (or your local env) with `main` 
2. Create a parent/child allocation pair and create files underneath the child allocation
3. Run Reset ACLs
4. Confirm that the parent group has been stripped for files in the child allocation and that `o` has been removed from the OWNER-associated permissions
5. Set up QA (or your local env) with this branch.
6. Run Reset ACLs
7. Confirm that the parent groups *do* appear on the files in the child allocation and that `o` still appears in permissions
8. Create a new parent/child allocation pair and repeat step 7.
